post_office/webhook.py

```diff
@@ -0,0 +1,117 @@
+"""
+A collection of utilities to help process webhook events from email services
+"""
+
+import json
+import logging
+from abc import ABCMeta, abstractmethod
+from enum import Enum
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse, HttpResponseNotFound
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+
+from .models import Email
+
+
+logger = logging.getLogger(__name__)
+
+
+class Event(Enum):
+    # Deliverability
+    ACCEPTED = 'accepted'
+    DELIVERED = 'delivered'
+    DEFERRED = 'deferred'
+    HARD_BOUNCE = 'hard_bounce'
+    SOFT_BOUNCE = 'soft_bounced'
```

Comment:
Should be `soft_bounce` instead of `soft_bounced`, to keep it consistent with `hard_bounce`.

Reply:
Fixed.